CONFIG_MPSL_CX disabled by default (#10259)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

  * nRF7002 revision A support.

* Updated:

>   * WiFi Coexistence is no longer enabled by default. It must be enabled explicitly in Kconfig by setting :kconfig:option:`CONFIG_MPSL_CX`. In case of nRF5340, this option must be selected for both application core and network core images.

Reviewer: ```suggestion
  * WiFi Coexistence is no longer enabled by default.
    It must be enabled explicitly in Kconfig using :kconfig:option:`CONFIG_MPSL_CX`.
    On the nRF5340, this option must be selected for both the application core and the network core images.
```
